app/uk/gov/hmrc/agentsubscriptionfrontend/controllers/package.scala

```diff
@@ -40,11 +42,13 @@ package object controllers {
       }
     }
 
-    private val nonEmptyTelephoneNumber: Constraint[String] = Constraint[String] { fieldValue: String =>
```

Comment:
Consistency is good, descriptive names are also good.
Have you implemented (this)[https://github.com/hmrc/agent-subscription-frontend/pull/50#discussion_r115237213] yet? You might end up changing what this constraint does whilst implementing that change, so may as well wait until after that change to decide whether to change the name of this constraint.